spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniprot/converter/UniProtEntryCommentsConverter.java

```diff
@@ -454,6 +454,15 @@ private Set<String> fetchEvidences(Comment comment) {
                             UniProtEntryConverterUtil.extractEvidence(
                                     caComment.getReaction().getEvidences()));
                 }
+                if (caComment.hasPhysiologicalReactions()) {
+                    var physiologicalEvidences =
```

Comment:
We should avoid using var. It reduces readability drastically and saves very little. We can discuss on this and read best practices.